react-ui/src/components/Challenges/self-monitor.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import Accordion from '../Accordion/accordion';
+import SocialIcons from '../SocialIcons/social-icons';
+import icon from '../../assets/Vitals.svg';
+
+
+const SelfMonitor = () => (
+    <Accordion
+        id="self-monitor"
+        title="Self-Monitor"
+        icon={icon}
+    >
+        <p>
+            Take care of yourself and reduce anxiety by using the PledgeToProtectME and GetWellNetwork COVID-19 Self-Monitoring Tool.
+        </p>
+        <p>
+            <b>It's simple, interactive, and sophisticated!</b>
+        </p>
+        
+        <a
+            href="https://apps.getwellnetwork.com/loop-enroll/pledgetoprotectme-covid/"
+            className="btn"
+            aria-label="Sign Up Here"
```

Comment:
FYI aria-label is not needed if the text for the link is already clear enough.  The text of the link is what will be read by screen readers so we only need to add an aria-label when that text doesn't provide enough context for those that can't see the page.